src/app/containers/AVPlayer/index.test.jsx

```diff
@@ -50,43 +50,33 @@ const AVPlayerCanonical = (
 const AVPlayerAMP = (
   <GenerateFixtureData
     platform="amp"
-    embedUrl="https://polling.test.bbc.co.uk/ws/av-embeds/media/pashto/bbc_pashto_tv/w172xcldhhrdqgb/ps/amp?morph_env=live"
+    embedUrl="https://polling.test.bbc.co.uk/ws/av-embeds/media/afaanoromoo/bbc_afaanoromoo_radio/w3ct0l8r/om/amp"
+    title="Audio Player"
+    type="audio"
+    iframeTitle="Audio player"
```

Comment:
Would we expect an English-language `iframeTitle` to be passed in here? The default for the component is translated, which is mirrored in the Canonical test?

Reply:
I don't know if it is a defect but in the afaanoromoo config, the translation for Audio Player is 'Audio Player' :)

Reply:
Well that's a real 'wat'.

⏰(closest thing to an alarm on this) @aeroplanejane is that expected?

Reply:
Sounds like some of the translations need updating in the config. See the spreadsheet pinned in #wstranslations on line   82. The translation for audio player in Afaan Oromoo is "Sagalee kan taphachiisu" for example.. but assuming this would probably warrant a separate ticket to update the missing translations?

Reply:
Can you guys point me to the translations in the config for the iframe and I'll draft an issue?

Reply:
https://github.com/bbc/simorgh/tree/latest/src/app/lib/config/services

https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/afaanoromoo.js#L54

Reply:
An enhancement here coulf be to dynamically access the values within that services' config file and pass those to `GenerateFixtureData` component, then hard code the assertion?

so something like passing `const iframeTitle = path(['translations', 'audioPlayer'] ,afaanOromooConfig) into `GenerateFixtureData`. 

At least then when you see a strange assertion passing - you see it's coming from a single source of truth.